examples/websub-hub-client-sample/publisher.bal

```diff
@@ -3,22 +3,23 @@
 import ballerina/io;
 import ballerina/runtime;
 import ballerina/websub;
+import ballerina/log;
 
-public function main() {
+public function main(string... args) {
 
     // Start up the internal Ballerina Hub.
     io:println("Starting up the Ballerina Hub Service");
 
     var result = websub:startHub(9191);
-    websub:WebSubHub webSubHub = result is error ? result.startedUpHub : result;
+    websub:WebSubHub webSubHub = result is websub:HubStartedUpError ? result.startedUpHub : result;
 
     // Register a topic at the hub.
     var registrationResponse = webSubHub.registerTopic(
                                             "http://websubpubtopic.com");
-    match (registrationResponse) {
-        error webSubError => io:println("Error occurred registering topic: "
-                                        + webSubError.message);
-        () => io:println("Topic registration successful!");
+    if (registrationResponse is error) {
+        log:printError("Error occurred registering topic:", err = registrationResponse);
```

Comment:
We usually use `io:println` with `main` files and `log` with services. Shall we stick to the original?

Reply:
Please check in all places.